### PR TITLE
Module1: portfolio

### PR DESCRIPTION
**Pull request by derrick1451**
in this project, l have created the first page of my portfolio website. it includes
1. an HTML file that contains the whole structure of the page
2. a CSS file that contains the styles markup
3. linter configuration file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title> myportfolio</title>
+</head>
+<body>
+    <header class="header-section">
+        <nav><a class="logo" href="#">DERRICK</a></nav>
+        <nav><img src="/images/toggle.png" alt=""></nav>
```
Comment (sja-thedude) [inline index.html:13]: - [ ] Kindly fix the logo according to the Figma design, it should move more to the left side.
- [ ] Kindly fix the hamburger button, it's not displayed.
- Your design:
Screenshot 2022-03-17 at 4 07 33 PM

- Figma design:
Screenshot 2022-03-17 at 4 06 31 PM

File: main.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    line-height: 20px;
+    text-decoration: none;
+    color: #6070FF;
+}
+.content {
+    display: flex;
+    flex-direction: column;
+    height: 90vh;
+    width: 90%;
+    background-image: url(/images/header.png);
```
Comment (sja-thedude) [inline main.css:32]: - Kindly fix the background image, it's not displayed.
Your design:
Screenshot 2022-03-17 at 4 02 36 PM

Figma design:
Screenshot 2022-03-17 at 4 03 40 PM

File: index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <section class="content">
+        <h1 class="intro-1">l'm Derrick glad to see you!</h1>
+        <P class="intro-2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</P>
+        <h3 class="intro-3">LET'S CONNECT</h3>
+        <ul class="socials">
+            <li><img src="/images/twitter.png" alt="twitter icon"></li>
+            <li><img src="/images/Linkedin.png" alt="linkedin icon"></li>
+            <li><img src="/images/m.png" alt="social icon"></li>
+            <li><img src="/images/github.png" alt="github icon"></li>
+            <li><img src="/images/fingers.png" alt="thumbsup"></li>
```
Comment (sja-thedude) [inline index.html:24]: - [ ] Kindly make sure that each `li` icon has a link `a tag`.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        </ul>
```
Comment (sja-thedude) [inline index.html:25]: - [ ] Kindly make sure your social links are displayed properly on the page.
- Your design:
Screenshot 2022-03-17 at 4 07 18 PM

- Figma design:
Screenshot 2022-03-17 at 4 19 40 PM

File: main.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+.intro-2 {
+height: 146px;
+width: 327px;
+left: 24px;
+top: 276px;
+position: static;
+color: #344563;
+line-height: 24px;
+margin: 12px 0;
+}
```
Comment (sja-thedude) [inline main.css:59]: - [ ] Make sure to follow the spacing, font family and font size according to the Figma design. Try to replicate the design as much as possible.
- Your design:
Screenshot 2022-03-17 at 4 02 36 PM

- Figma design:
Screenshot 2022-03-17 at 4 03 40 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @derrick1451, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional Readme 👍 
- Good commit messages 👍 

## Required Changes ♻️
- Kindly fix the linter error.
- Make sure to add a descriptive title and summary of the pull request.
- _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.css [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  justify-content: space-between;
+  align-items: center;
+  padding: 24px;
+  width: 100%;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  background-image: url(/images/Header\ bg@2x.png);
```
Comment (Donard97) [inline main.css:36]: - [ ] You did an amazing job implementing requested changes from the other reviewer but please stick as much as you can to the [Figma design](). Keep rocking 💪 

Here you have a screenshot from your project:

image

Here you have a screenshot from the design:

image

File: README.md [hunk 7]
```diff
@@ -1 +1,64 @@
...
+
+<!-- ## Live Demo (if available)q
+
+[Live Demo Link](https://livedemo.com) -->
+
+
+<!-- ## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.** -->
```
Comment (Donard97) [inline README.md:20]: - [OPTIONAL] Kindly uncomment the getting start section and please let us know how to clone, setup, and run your project and please delete the unused section. Keep coding 💪

**Review by Donard97 — CHANGES_REQUESTED** [review]
Hi @derrick1451 , 👋  
    
_Great work on making the changes requested by a previous reviewer 👏 💯_ 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 💪 

giphy (5)


## Highlights 🏁 

- There is no linter error ✔️ 
- Gitflow is used correctly ✔️ 
- PR is descriptive ✔️ 
    
## Required Changes ♻️

- [ ] Please check the comments under the review._
    
## Optional suggestions

- [OPTIONAL] I suggest you in the future to give more meaningful names to the branches for example you have to name in base what you will do on that branch. Keep shining 💪 
    
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
    
**Cheers and Happy coding!👏👏👏**
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
    
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
    
------
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
 ```

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

## *** Hi @derrick1451 *** , 



### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**